load-tests/package.json

```diff
@@ -13,7 +13,6 @@
   "author": "Microsoft",
   "license": "Apache-2.0",
   "dependencies": {
-    "@types/node": "^20.14.2",
```

Comment:
This was for type checking the code, is this not necessary anymore?

For type checking inside vscode for example.